keps/sig-cluster-lifecycle/0004-bootstrap-checkpointing.md

```diff
@@ -1,29 +1,27 @@
 # Kubernetes Bootstrap Checkpointing Proposal
 
 ## Metadata
-
-```yaml
-kep-number: draft-20171029-boostrap-checkpointing.md
+```
+---
+kep-number: 4
 title: Kubernetes Bootstrap Checkpointing Proposal
+status: implemented
 authors:
-  - name: Timothy St. Clair
-    github: @timothysc
-    email: timothysc@gmail.com
+  - "@timothysc"
 owning-sig: sig-cluster-lifecycle 
-participating-sigs:
-  - sig-node
+participating-sigs: sig-node
```

Comment:
This should be in list format, e.g.

```
participating-sigs:
  - sig-node
```

Reply:
I don't know... @jbeda?

Reply:
Yeah -- these should be a list.  We should create a presubmit check that validates the schema in front of each KEP.